pkg/machine/ignition_linux.go

```diff
@@ -1,12 +1,22 @@
 package machine
 
 import (
+	"os"
 	"os/exec"
 	"strings"
 )
 
 func getLocalTimeZone() (string, error) {
-	output, err := exec.Command("timedatectl", "show", "--property=Timezone").Output()
+	var output []byte
+	var err error
+
+	_, lookupErr := exec.LookPath("timedatectl")
+	if lookupErr != nil {
+		output, err = os.ReadFile("/etc/timezone")
+	} else {
+		output, err = exec.Command("timedatectl", "show", "--property=Timezone").Output()
```

Comment:
Why lookpath at all, why not attempt to exec.Command and if it fails with ENOEXISTS attempt to ReadFile?

Reply:
> Why lookpath at all, why not attempt to exec.Command and if it fails with ENOEXISTS attempt to ReadFile?

Thank you for your reviewing.

It is because both were thought to be equivalent due to platforms. Also, few nesting of code.

Well, I'll try to impl to check the error first.

Reply:
Update:

I checked if I could catch `ErrNotExist`. Both of the below were false.
In addition, [os/exec](https://pkg.go.dev/os/exec#Error) doesn't seem to return the Error type ?

```go
package main

import (
	"fmt"
	"os"
	"os/exec"
	// "errors"
)

func main() {
	_, err := exec.Command("timedatectl").Output()
	x := os.IsNotExist(err)
	// x := errors.Is(err, os.ErrNotExist)
	fmt.Printf("%v", x)
	// false
}
```

Reply:
```
	if output, err = exec.Command("timectl", "show", "--property=Timezone").Output(); errors.Is(err, exec.ErrNotFound) {
		output, err = os.ReadFile("/etc/timezone")
	}
```

Reply:
> ```
> 	if output, err = exec.Command("timedatectl", "show", "--property=Timezone").Output(); errors.Is(err, exec.ErrNotFound) {
> 		output, err = os.ReadFile("/etc/timezone")
> 	}
> ```

I'm so sorry. I have lots to learn.
Thank you for your really kind suggestion (almost the whole).

I sent PR again.
The re-tests were successful: Fedora 37 (systemd) and Artix (systemd-free).